src/js/components/Main Components/Breeds/Subcomponents/Information/BaseInformation/baseInformation.js

```diff
@@ -20,20 +20,20 @@ function BaseInformation(props) {
                 </p>
             </div>
             <div className={'description'}>
-                <span className={'bold'}>Description: </span>{selectedBreed.description}
+                <p><span className={'bold'}>Description: </span>{selectedBreed.description}</p>
```

Comment:
Jedna uwaga...jak masz propsy w JSX...i ten props jest tekstowy to nie potrzebnie piszesz wszędzie `={'bold'}`

Wystarczyłoby samo `="bold"`